### ISSUE TITLE
Globalization: Emoji description

### ISSUE THREAD

**Issue by WilliamHolmes** [twitter/twemoji#246]
Are there any plans to globalize the emojis title/alt description?

i.e. `grinning face`, `face with open mouth` etc.

**Comment by Crissov** [comment]
Twemoji could import [CLDR data](http://unicode.org/cldr/charts/latest/annotations/index.html).

**Comment by WebReflection** [comment]
> Are there any plans to globalize the emojis title/alt description?

Not that I am aware of. Few possible reasons:

  * one does not simply import CLDR data
  * the description of each emoji, combined in a script, is heavy/huge. One benefit of this library is that it's tiny
  * the description of each emoji is not necessarily English. Solving all possible language language issues in here doesn't seem to be the right place
  * for a11y purpose, I'm pretty sure readers already have that info
  * for everyone else, emoji should already mean something
  * since latter point, not many hold in top of an emoji waiting for _smiling face_ info

Last, but not least, if it's about helping people writing down emoji on text-areas there are already projects that map English emoji names to the right image, using Twemoji library.

TL;DR I don't think what you are asking should land in this repository. Eventually I would consider a PR that creates a new file with such functionality, without touching current one (i.e. `2/temoji-named.js`) but I'm busy with my own stuff and I don't have much time for that ( I don't work at twitter, nobody pays me here ).

Best Regards
